backends/aptcc/acqpkitstatus.cpp

```diff
@@ -79,6 +79,13 @@ void AcqPackageKitStatus::Fetch(pkgAcquire::ItemDesc &Itm)
 /* We don't display anything... */
 void AcqPackageKitStatus::Done(pkgAcquire::ItemDesc &Itm)
 {
+    PkRoleEnum role = pk_backend_job_get_role(m_job);
+    if (role == PK_ROLE_ENUM_REFRESH_CACHE) {
+        pk_backend_job_repo_detail(m_job,
+                                   "",
```

Comment:
I think you want a repo ID here...

Reply:
Yeah, technically... unfortunately we don't have easy access to it. So I left that intentionally broken same as in `AcqPackageKitStatus::IMSHit`.

What apt sends there are basically protocol requests succeeding/failing, but in the one minute I spent digging through apt-pkg's API I didn't find a way to get from there to the information we need to build a repo id.

There **may** be a way to infer it (break URI apart, compare to possible URI parts a repo might feasibly have). But it's out of scope for what I actually want to fix: the current detail list isn't even complete.